recipe/meta.yaml

```diff
@@ -23,36 +23,46 @@ requirements:
   run:
     - python >=3.6
     - numpy
-    - scipy
+    - scipy >=1.2.0
     - scikit-image
-    - vispy
-    - dask
-    - cachey
+    - vispy >=0.6.4
+    - dask[array] >=2.1.0
+    - cachey >=0.2.1
     - zarr
     - fsspec
-    - imageio
-    - ipykernel
+    - imageio >=2.5.0
+    - importlib-metadata >=1.5.0
+    - ipykernel >=5.1.1
+    - IPython >=7.7.0
     - jupyter_client
-    - qtpy
-    - qtconsole
+    - qtpy >=1.7.0
+    - qtconsole >=4.5.1
     - IPython
     - backcall
+    - Pillow !=7.1.0,!=7.1.1  # not a direct dependency, but 7.1.0 and 7.1.1 broke imageio
     - pluggy
-    - PyOpenGl
+    - psutil >=5.0
+    - PyOpenGL >=3.1.0
     - PySide2
-    - wrapt
-    - numpydoc
-    - napari-plugin-engine
-    - napari-svg
-    - tifffile
+    - PyYAML >=5.1
+    - wrapt >=1.11.1
+    - numpy >=1.10.0
+    - numpydoc >=0.9.2
+    - napari-plugin-engine >=0.1.5
+    - napari-svg >=0.1.3
+    - tifffile >=2020.2.16
+    - toolz >=0.10.0
```

Comment:
Does `napari` need `toolz`? Asking as this appears to not have been listed previously.

Reply:
I think it does, see [here](https://github.com/napari/napari/blob/ab741703a2e0457abf69337bb7b15a39b2a39216/napari/layers/transforms.py#L1) and [here](https://github.com/napari/napari/blob/af011f9a78a13d46f0d67bd0c7106504243683e9/napari/_qt/threading.py#L6)